### ISSUE TITLE
Optimization 실패 Pass의 경우 질문

### ISSUE THREAD

**Issue by sungyub1399** [snu-sf-class/swpp202101#83]
안녕하세요 조교님,

다름이 아니라 Optimization을 진행하여 코드를 작성했으나, 컴파일은 잘 됐고 실제로 optimization은 실패했습니다.
해당 경우 PR을 진행하여 master에 squash merge를 하는 게 좋은지 아니면 그냥 PR을 close 하고 머지를 안 하는 것이 좋은지 여쭤보고 싶습니다.

감사합니다

**Comment by Sung-HwanLee** [comment]
최적화를 구현하였으나, 실제 벤치마크에서 해당 최적화가 일어나지 않았다는 말씀이신가요?
merge하여도 문제는 없습니다만, 성능향상에 도움이 될지에 대해서는 학생분들께서 판단해주셔야 할 것 같습니다.

**Comment by sungyub1399** [comment]
> 최적화를 구현하였으나, 실제 벤치마크에서 해당 최적화가 일어나지 않았다는 말씀이신가요?
> merge하여도 문제는 없습니다만, 성능향상에 도움이 될지에 대해서는 학생분들께서 판단해주셔야 할 것 같습니다.

네 맞습니다. 벤치마크에서 최적화가 일어나지 않았습니다. 조금 더 고민을 하고 merge 할지 하지 않을지 결정하겠습니다 답변 감사합니다!!
